### PR TITLE
Add menu tips article

### PR DESCRIPTION
**Pull request by Walavouchey**
closes https://github.com/ppy/osu-wiki/issues/7971

not entirely sure if "menu tips" is encompassing enough but it sounds fairly ok to me, and they're mostly about menus (client navigation and shortcuts) anyway

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Comment by TicClick** [comment]
somehow an article full of large lists alone doesn't look right to me. I mean, I don't have anything against it (it's informative, scratches an itch, etc, etc), but it could be a few lines on the `Client` article pointing to locations of the tips as well...

while I don't have any suggestions for its core design, because well, lists, how about you add some minimal formatting (for example, `Hot`+`Key`+`Combinations`) and links to other articles where appropriate to add more value to it?

**Comment by peppy** [comment]
Yeah, I'm not sure of the value provided. For stable it's probably fine because I don't foresee the tips changing, but for lazer, it might be just as beneficial to link to the code file with the tips present (so they don't have to be updated in multiple places)?

**Comment by Walavouchey** [comment]
if it's about value, then the suggestions ticclick provided would be the answer. perhaps we could even add illustrative gifs for better presentability.

the reason this is put in an article in the first place is to have an accessible, searchable source of information. sure, this could be a stub article that links to the lazer source and tells people to check the stable localisation files, but then we wouldn't have much of an article.

also, it's not like the same thing would make sense for other articles with duplicate and manually updated information; much of the information on the wiki is a secondary source anyway, requiring updates as time passes (pp, medals, community articles, all other lists, etc, not to mention new info regarding lazer). the wiki is not static, at least for as long as the game and the community evolve.

File: wiki/Client/Menu_tips/en.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+## Main menu
+
+These tips appear at the bottom of the [main menu](/wiki/Client/Interface#main-menu).
+
+- Add new [beatmaps](/wiki/Beatmap) to osu! by simply dragging the [`.osz` file](/wiki/Client/File_formats/Osz_(file_format)) into the window, double-clicking it in Windows Explorer, or choosing "open" when downloading in your browser.
+- You can efficiently navigate large lists of beatmaps in [song select](/wiki/Client/Interface#song-select) by holding left-mouse to drag the list, or right-mouse to quick-seek.
+- Press `F8`/`F9` anywhere in game to join osu! [chat](/wiki/Client/Interface/Chat_console)!
+- Right-clicking on a beatmap or clicking the "Beatmap Options" button in the song select will show a menu that will allow you to manage the beatmap.
+- If your game or mouse feels laggy, try toggling the `Framerate Limit` in [Options](/wiki/Client/Options) or using `Fullscreen mode`/`Letterboxing`! It might just help!
```
Comment (Walavouchey) [inline wiki/Client/Menu_tips/en.md:25]: nowadays called "Frame limiter"

File: wiki/Client/Menu_tips/en.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+- Make sure to take breaks every so often. Your hands have to serve you in life for more than just osu! [spinners](/wiki/Gameplay/Hit_object/Spinner).
+- Don't ignore [unranked](/wiki/Beatmap/Category) maps. If you find some you like and leave constructive comments, there is a much higher chance of them being [ranked](/wiki/Beatmap_ranking_procedure)!
+- You can play songs together with other players in [multiplayer mode](/wiki/Client/Interface/Multiplayer)!
+- Ever wondered where someone was from? Hit `F9` and hover their user panel!
+- Hit `F5` at the song select screen to refresh and load any new songs from disk.
+- Hit `Shift` + `F12` anywhere in-game to take a screenshot and automatically upload it to the internet.
+- You can customise the keys you use to play osu! under `Options` > `Input`.
+- You can search for [specific conditions](/wiki/Client/Interface#search) at song select, like "ar>8 od=9". Supported keywords are: `ar`, `od`, `hp`, `cs`, `bpm`, `length` and `drain` (time in seconds).
```
Comment (Walavouchey) [inline wiki/Client/Menu_tips/en.md:49]: there are way more filters supported according to https://osu.ppy.sh/wiki/en/Client/Interface#search

File: wiki/Client/Menu_tips/en.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+## Editor
+
+These tips appear in the [beatmap editor](/wiki/Client/Beatmap_editor) below the [hit object timeline](/wiki/Client/Beatmap_editor/Timelines#hit-objects).
+
+- Use the keyboard shortcuts `1`, `2`, `3`, and `4` to quickly navigate between different [object types](/wiki/Gameplay/Hit_object). As a mapper, you'll be using these buttons all the time!
+- Right-clicking on objects in the playfield or timeline will remove them. Right-clicking on an empty spot of the playfield will make the next object or any selected objects the start of a new [combo](/wiki/Beatmapping/Combo).
+- Clicking while drawing a [slider](/wiki/Gameplay/Hit_object/Slider) creates a curvy gray [anchor](/wiki/Gameplay/Hit_object/Slider/Slider_anchor) and double-clicking creates a sharply angled red anchor. `Ctrl` + right-clicking adds anchors to completed sliders.
+- Timing is important! Α wrongly timed beatmap is frustrating to map and play. If you need timing assistance, post your beatmap in the Timing Help thread of the [Modding Queues forum](https://osu.ppy.sh/community/forums/60) or ask for a timing check in `#modreqs`.
```
Comment (Walavouchey) [inline wiki/Client/Menu_tips/en.md:59]: what is this "Timing Help" thread

**Review by Walavouchey — COMMENTED** [review]
some of these are very outdated, maybe they should be updated in-game?

File: wiki/Client/Menu_tips/en.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+- For fast combos, try toggling between left and right mouse button, or use keyboard alternative `Z`/`X`!
+- If you can't play videos, take a look at the wiki before posting a bug report!
+- Take a screenshot from within osu! by hitting `F12`.
+- Try watching others as they play using [spectator mode](/wiki/Gameplay/Spectating). Hit `F9` to toggle the extended chat interface and utilise this!
+- When a beatmap has been updated, as long as you are in online ranking mode you will be given a button to update to the latest version.
+- The wiki has a wealth of information in it. Seriously, read it.
+- Did you know you can load new beatmaps by simply double-clicking on an `osz` file or dragging it into a running osu! window?
+- You can export any replay whether it be online or offline by hitting `F2` at the [ranking screen](/wiki/Client/Interface#results-screen).
+- If you find that the 'beats' sound too early or late in every beatmap you play, try adjusting the `Universal Offset` in `Options`.
+- The fastest way to find a beatmap is to simply type part of its title/artist/creator directly into the song select screen.
```
Comment (TicClick) [inline wiki/Client/Menu_tips/en.md:35]: has useful tips on search syntax
```suggestion
- The fastest way to find a beatmap is to simply type part of its title/artist/creator directly into the [song select screen](/wiki/Client/Interface#song-select).
```

File: wiki/Client/Menu_tips/en.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+- Adjust the volume in-game using the mouse wheel. This can be disabled in options if you wish.
+- Pause/unpause the game quickly by hitting the middle mouse button. This can be disabled in options if you wish.
+- You can gain quick access to web links by clicking the chat bubble icon in song select, just above the [score listings](/wiki/Client/Interface#rankings).
+- Customise your osu! experience with new [skins](/wiki/Skin)! Download them from the [Skinning forum](https://osu.ppy.sh/community/forums/15).
+- Hit `F8` at any time to activate the in-game chat window. You can check out the [commands availablei](/wiki/Client/Interface/Chat_console#commands-list) in chat by typing `!help` or `/help`
+- Make sure to take breaks every so often. Your hands have to serve you in life for more than just osu! [spinners](/wiki/Gameplay/Hit_object/Spinner).
+- Don't ignore [unranked](/wiki/Beatmap/Category) maps. If you find some you like and leave constructive comments, there is a much higher chance of them being [ranked](/wiki/Beatmap_ranking_procedure)!
```
Comment (TicClick) [inline wiki/Client/Menu_tips/en.md:43]:
```suggestion
- Don't ignore [unranked](/wiki/Beatmap/Category) maps. If you find some you like and leave [constructive comments](/wiki/Modding), there is a much higher chance of them being [ranked](/wiki/Beatmap_ranking_procedure)!
```

File: wiki/Client/Menu_tips/en.md (hunk 2 above)
Comment (TicClick) [inline wiki/Client/Menu_tips/en.md:49]:
```suggestion
- You can search for [specific conditions](/wiki/Client/Interface#search) at song select, like `ar>8 od=9`. Supported keywords are: `ar`, `od`, `hp`, `cs`, `bpm`, `length` and `drain` (time in seconds).
```

File: wiki/Client/Menu_tips/en.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+- You can customise the keys you use to play osu! under `Options` > `Input`.
+- You can search for [specific conditions](/wiki/Client/Interface#search) at song select, like "ar>8 od=9". Supported keywords are: `ar`, `od`, `hp`, `cs`, `bpm`, `length` and `drain` (time in seconds).
+- Quickly retry a beatmap by holding `Ctrl` + `R` for a short time! You can also use a custom key (see `Quick Retry` in `Keyboard Bindings`).
+
+## Editor
+
+These tips appear in the [beatmap editor](/wiki/Client/Beatmap_editor) below the [hit object timeline](/wiki/Client/Beatmap_editor/Timelines#hit-objects).
+
+- Use the keyboard shortcuts `1`, `2`, `3`, and `4` to quickly navigate between different [object types](/wiki/Gameplay/Hit_object). As a mapper, you'll be using these buttons all the time!
+- Right-clicking on objects in the playfield or timeline will remove them. Right-clicking on an empty spot of the playfield will make the next object or any selected objects the start of a new [combo](/wiki/Beatmapping/Combo).
```
Comment (TicClick) [inline wiki/Client/Menu_tips/en.md:57]:
```suggestion
- Right-clicking on objects in the playfield or [timeline](/wiki/Client/Beatmap_editor/Timelines#hit-objects) will remove them. Right-clicking on an empty spot of the playfield will make the next object or any selected objects the start of a new [combo](/wiki/Beatmapping/Combo).
```

File: wiki/Client/Menu_tips/en.md (hunk 3 above)
Comment (TicClick) [inline wiki/Client/Menu_tips/en.md:59]:
```suggestion
- Timing is important! Α wrongly timed beatmap is frustrating to map and play. If you need timing assistance, post your beatmap in the [Timing Help thread](https://osu.ppy.sh/community/forums/topics/13795) of the [Modding Queues forum](https://osu.ppy.sh/community/forums/60) or ask for a timing check in `#modreqs`.
```

File: wiki/Client/Menu_tips/en.md [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+- Swap the start and end positions of a slider with the "reverse selection" tool (`Ctrl` + `G`). Reversing the selection of multiple objects switches their positions in the timeline as well.
+- [Jumps](/wiki/Beatmap/Pattern/Jump) can be used to highlight important sounds in the music!
+- [Distance snap](/wiki/Client/Beatmap_editor/Distance_snap) is a handy tool for precise spacing calculations. Hold the `Alt` key to fine-tune it.
+- Try synchronising the most important sounds in a song to clicks in gameplay. This is especially important when deciding slider [rhythms](/wiki/Music_theory/Rhythm)!
+- The [`Beat Snap Divisor`](/wiki/Client/Beatmap_editor/Beat_Snap_Divisor) option in the top right determines how objects synchronise to music. For most songs, sticking to the default `1/4` value will work fine.
+- If you're looking for feedback on your map, check out the Modding Queues forum, the `#modreqs` channel in-game, or ask your friends for advice!
+- Settings like HP drain, Circle Size, and Approach Rate can be adjusted in the `Difficulty` tab of `Song Setup` (`F4`).
```
Comment (TicClick) [inline wiki/Client/Menu_tips/en.md:67]: may be worth it to link respective pages here (HP/CS/AR/Song setup)

File: wiki/Client/Menu_tips/en.md [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+- Bookmarks (`Ctrl` + `B`) are a fantastic tool to visualise a song's changes. Many mappers separate large sections of music with bookmarks to easily coordinate changes in their mapping.
+- Try reading the site's wiki, asking a question in `#modhelp`, or posting on the Beatmap Help forum if you need additional mapping information!
```
Comment (TicClick) [inline wiki/Client/Menu_tips/en.md:69]:
```suggestion
- Try reading the site's wiki, asking a question in `#modhelp`, or posting on the [Beatmap Help forum](https://osu.ppy.sh/community/forums/56) if you need additional mapping information!
```

File: wiki/Client/Menu_tips/en.md [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+- [Test your beatmap](/wiki/Client/Beatmap_editor/Test_mode) (`F5`) to identify potential gameplay problems and improvements.
+- Disable [`Grid Snapping`](/wiki/Beatmapping/Grid_snapping) if you want unhindered control of object composition. You can be extra precise by selecting objects and using the `Ctrl` + Arrow key shortcuts too.
+- When your beatmap is ready for feedback, you can upload it through the Beatmap Submission System! Look for the `Upload Beatmap` option under the `File` dropdown menu.
```
Comment (TicClick) [inline wiki/Client/Menu_tips/en.md:72]:
```suggestion
- When your beatmap is ready for feedback, you can upload it through the [Beatmap Submission System](/wiki/Beatmapping/Beatmap_submission)! Look for the `Upload Beatmap` option under the `File` dropdown menu.
```

File: wiki/Client/Menu_tips/en.md [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+- Geometric object arrangements can be made with the "[Create Polygon Circles](/wiki/Client/Beatmap_editor/Menu#special-commands.2)" tool (`Ctrl` + `Shift` + `D`).
+- Beats placed on top of each other will be [stacked](/wiki/Beatmapping/Mapping_techniques/Stack) in play mode. To see how stacks look during the play, select `Stacking` from the `View` dropdown menu.
+- Using [AiMod](/wiki/Client/Beatmap_editor/AiMod) (`Ctrl` + `Shift` + `A`) is a fantastic way to identify potential issues with your map long before the first [modder](/wiki/Modding/Modder) ever checks it out.
+- Navigate the timeline by scrolling or using the left/right arrow keys. Hold `Shift` to navigate 4x faster.
+- You can use the copy/paste commands (`Ctrl` + `C`/`V`) to duplicate objects in your beatmap. In conjunction with horizontal/vertical flipping (`Ctrl` + `H`/`J`) and rotation (`Ctrl` + `Shift` + `R`), you can easily arrange objects into cute patterns!
+- Use the playback speed buttons in the bottom right to slow the song down and easily decipher timing and rhythms.
+- Check out the Featured Artist listing if you're looking for new pre-timed songs to map!
```
Comment (TicClick) [inline wiki/Client/Menu_tips/en.md:83]:
```suggestion
- Check out the [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists) if you're looking for new pre-timed songs to map!
```

**Review by TicClick — CHANGES_REQUESTED** [review]
suggestions mostly

File: wiki/Client/Menu_tips/en.md (hunk 1 above)
Comment (TicClick) [reply wiki/Client/Menu_tips/en.md:25]: may be worth a footnote here

**Review by TicClick — APPROVED** [review]

**Pull request merged**